apps/occurrences/serializers.py

```diff
@@ -8,13 +8,13 @@
 class OccurrenceSerializer(SearchableModelSerializer):
     """Serializer for occurrences."""
 
-    date_html = serpy.Field()
+    dateHtml = serpy.Field(attr='date_html')
     summary = serpy.Field(attr='summary.html')
     description = serpy.Field(attr='description.html')
     postscript = serpy.MethodField()
-    serialized_images = serpy.Field()
-    primary_image = serpy.Field()
-    serialized_citations = serpy.Field()
+    serializedImages = serpy.Field(attr='serialized_images')
+    primaryImage = serpy.Field(attr='primary_image')
+    serializedCitations = serpy.Field(attr='serialized_citations')
```

Comment:
Is it safe to update the case on these as well?
https://github.com/ModularHistory/modularhistory/blob/da66384674d97e0ea2f436b57a313b1ed6d7d184/apps/search/models/searchable_model.py#L65-L73

Reply:
Yep!